src/raftstore/store/peer.rs

```diff
@@ -513,6 +532,21 @@ impl Peer {
         Ok(())
     }
 
+    fn use_local_read(&self, req: &RaftCmdRequest) -> bool {
+        if !req.has_header() || !req.get_header().has_read_quorum() ||
```

Comment:
`req.has_header() || !req.get_header().has_read_quorum() || req.get_header().get_read_quorum()` can be collapsed as `req.get_header().get_read_quorum()`.